bosun.md

```diff
@@ -0,0 +1,44 @@
+# Steering Committee Bosun Guide  
+
+This doc is a guide to help the delegated Steering Committee (SC) Member bosun the public meeting. SC members will pick a new one for each meeting.  
+
+## Before
+1. Create the skeleton meeting agenda from follow up items the meeting prior, parking lot topics at the top, and ask other steering members to populate.
+2. Read through the agenda just to familiarize yourself with the content to assign timeboxes. [repeat a few times in the days leading to the meeting if it’s a live agenda]
+3. Post the draft agenda at least 3 days before with a call for final topics and a clear timebox to the kubernetes-dev mailing list. This will act as a reminder and put the agenda on the community’s radar, too.
```

Comment:
nit: also to the steering mailing list?

Reply:
3 business days

I was thinking something like: e-mail out wednesday by noon PT, close agenda friday by noon PT, hold meeting monday